### PR TITLE
Extension + Doc changes for referencing queries without having to reference the `data` object

### PR DESCRIPTION
**Pull request by ud3sh**
Took a quick look at how the docs would need to change to reflect that queries can be referenced directly.  It was mostly a search and replace so I just opened a PR (with the exception of sql-queries.md).  Opening this for feedback.

### CODE REVIEW

File: sites/docs/docs/features/queries/sql-queries.md [hunk 1]
```diff
@@ -11,17 +11,21 @@ When you open a page in dev mode, Evidence runs all of the queries on the page.
...
+All query results on a page are returned to a single object called `data`. To use a query result, you can to reference  a subset of the `data` object. For instance if your query is named `sales_by_country`, you can reference the results using `data.sales_by_country`.  For convinience, Evidence also exposes each query result by it's own name, e.g `sales_by_country`.  These references can be used in any of the components from our built-in library.
 
 For example, if your query name was `sales_by_country`:
 ```markdown
+<LineChart data={sales_by_country}/>
+```
+or alternatively,
+```markdown
 <LineChart data={data.sales_by_country}/>
 ```
```
Comment (archiewood) [inline sites/docs/docs/features/queries/sql-queries.md:31]: @udesh I think this is a bit confusing. 
I can't really imagine a user ever really using this {data.query} functionality, unless they were very advanced.

I'd replace line 21 with:
```
# Using Query Results 
Reference a query in a component using `data={query_name}`

For example, if your...
```

**Comment by archiewood** [comment]
Love how much cleaner all this code is

File: sites/docs/docs/features/queries/sql-queries.md (hunk 1 above)
Comment (ud3sh) [reply sites/docs/docs/features/queries/sql-queries.md:31]: Makes sense to me.  Fixed as suggested.

**Comment by ud3sh** [comment]
> Yes all the core functionality works well in the tutorial project, components, loops, {#each} etc - when you merge this I will merge an update there also.

Thanks for testing it out @archiewood !

**Comment by hughess** [comment]
I just remembered that we should also change the snippets in the VS Code extension to remove “data.”

Does that make sense to do in here or should I open a new PR for that?

Great job on this syntax change - it’s fantastic, I’m so excited for this.

**Comment by ud3sh** [comment]
> I just remembered that we should also change the snippets in the VS Code extension to remove “data.”
> 
> Does that make sense to do in here or should I open a new PR for that?
> 
> Great job on this syntax change - it’s fantastic, I’m so excited for this.
@hughess Good callout.  If this is it, we can do it under the same PR
https://github.com/evidence-dev/evidence/blob/ce3fcddc47a380d6dafb4a692b05d276db4ead7a/packages/evidence-vscode/snippets/emd.code-snippets

**Pull request merged**